src/components/list/video-list/VideoList.module.css

```diff
@@ -0,0 +1,5 @@
+.VideoList {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(30rem, 1fr));
+  padding: 4rem;
+}
```

Comment:
suggestion - you can add `justify-items: center` to the cards container so that they remain on the center.

reference - https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items